SCSS/README.md

```diff
@@ -0,0 +1,219 @@
+<p align="center">
+   <img src="https://assets1.unidays.world/v5/main/assets/images/logo_v003.svg" width="60%"/>
+ </p>
+<br/>
+
+# UNiDAYS SCSS Coding Style Guide
+
+[SCSS File Naming Convention](#scss-file-naming-convention)
+
+[SCSS Code Styles](#scss-code-styles)
+  * [Class Naming Conventions](#class-naming-conventions)
+  * [Modifiers](#modifiers)
+  * [Formatting](#formatting)
+  * [Nesting](#nesting)
+  * [Avoid Tag Selectors](#avoid-tag-selectors)
+  * [Mixins Extends and Placeholders](#mixins-extends-and-placeholders)
+  * [Colour Palette](#colour-palette)
+
+
+### SCSS File Naming Convention
+
+All files should be named using kebab-case where applicable.
+For example: `c-component-name.scss`
+
+File names should be the same name of the parent class within that file.
+
+For example:
+`c-menu-nav.scss`
+```
+.c-menu-nav
+{
+	//styles here
+}
+```
+
+
+
+
+### SCSS Code Styles
+
+#### Class Naming Conventions
+Think of your CSS in a component based way and make your classes as reusable as possible. 
+
+Prefix class names according to the following:<br> 
+`.t-` typography styles<br>
+This relates to any typography specific styles for headers or body copy. 
+
+`.c-` component styles<br>
+This relates to component specific styles. Aim to make these as reusable as possible. 
+Avoid adding layout specific styles to these unless these styles will never be changed. For example avoid adding set margins and floats etc as these might need to be changed according to the layout these components live in. 
+
+`.l-` layout styles<br>
+These classes are used to define the layout of pages and determine how and where the components and text elements sit within the page.  
+
+#### Modifiers
+
```

Comment:
If this style guide is also being used by those less familiar with FE / SASS - maybe add either a link to what a modifier is, or a short explanation?

Reply:
Added more detail to this.